# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: layout/generic/nsBlockFrame.cpp
# Commit: 8647528a4e9d
# Full Hash: 8647528a4e9d6350ba9b1ce6245d51476081c5e4
# Author: Mats Palmgren <mats@mozilla.com>
# Date: 2019-08-14 21:57:52
# Regressor Bug: 1105868
# File Overlap Count: 2
# Description:
#   Bug 1105868 part 4 - Accessibility support for inline list-items.  r=emilio
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D39833
# ==============================================================================

diff -r b517c3b51b9a -r 8647528a4e9d layout/generic/nsBlockFrame.cpp
--- a/layout/generic/nsBlockFrame.cpp	Wed Aug 14 14:37:03 2019 +0000
+++ b/layout/generic/nsBlockFrame.cpp	Wed Aug 14 14:37:16 2019 +0000
@@ -24,7 +24,6 @@
 #include "nsAbsoluteContainingBlock.h"
 #include "nsBlockReflowContext.h"
 #include "BlockReflowInput.h"
-#include "nsBulletFrame.h"
 #include "nsFontMetrics.h"
 #include "nsGenericHTMLElement.h"
 #include "nsLineBox.h"
@@ -66,8 +65,6 @@
 
 static const int MIN_LINES_NEEDING_CURSOR = 20;
 
-static const char16_t kDiscCharacter = 0x2022;
-
 using namespace mozilla;
 using namespace mozilla::css;
 using namespace mozilla::dom;
@@ -7114,30 +7111,6 @@
          marker->StyleContent()->ContentCount() == 0;
 }
 
-#ifdef ACCESSIBILITY
-void nsBlockFrame::GetSpokenMarkerText(nsAString& aText) const {
-  const nsStyleList* myList = StyleList();
-  if (myList->GetListStyleImage()) {
-    aText.Assign(kDiscCharacter);
-    aText.Append(' ');
-  } else {
-    if (nsIFrame* marker = GetMarker()) {
-      if (nsBulletFrame* bullet = do_QueryFrame(marker)) {
-        bullet->GetSpokenText(aText);
-      } else {
-        ErrorResult err;
-        marker->GetContent()->GetTextContent(aText, err);
-        if (err.Failed()) {
-          aText.Truncate();
-        }
-      }
-    } else {
-      aText.Truncate();
-    }
-  }
-}
-#endif
-
 void nsBlockFrame::ReflowOutsideMarker(nsIFrame* aMarkerFrame,
                                        BlockReflowInput& aState,
                                        ReflowOutput& aMetrics,